src/app/large-display/large-display.component.ts

```diff
@@ -111,34 +153,107 @@ export class LargeDisplayComponent implements OnInit, AfterViewInit {
     }
   ];
 
-  running: boolean;
+  currentScreen: ScreenId = ScreenId.WaitingToAuthenticate;
+
+  running$: Observable<boolean> = this.timerService.running$.pipe(
+    // We need the explicit type parameters or else tslint can't tell which
+    // overload we're using, and it mistakenly thinks we're invoking the
+    // function in a deprecated manner.
+    startWith<boolean, null>(null),
+    shareReplay(1),
+  );
```

Comment:
Why are we setting this up in the component? Is there something extra needed that `running$` from the timer service doesn't provide?

Reply:
Oh, I didn't realize `TimerService.running$` had a `shareReplay()` on it! Never mind then :) Thanks!